### ISSUE TITLE
gtk-demo error: Builder

### ISSUE THREAD

**Issue by kojix2** [ruby-gnome2/ruby-gnome2#1175]
gtk-demo error report

```bash
/ruby-gnome2/gtk3/sample/gtk-demo$ ruby main.rb
```

Builder sample crashed. This is the error message.

```bash
Gtk-ERROR **: failed to add UI: .:82:98 Unknown flag: 'primary'
	from .rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/gobject-introspection-3.2.5/lib/gobject-introspection/loader.rb:317:in `block (2 levels) in load_constructor_infos'
	from .rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/gtk3-3.2.5/lib/gtk3/builder.rb:74:in `initialize'
	from /ruby-gnome2/gtk3/sample/gtk-demo/builder.rb:11:in `new'
	from /ruby-gnome2/gtk3/sample/gtk-demo/builder.rb:11:in `initialize'
	from main.rb:181:in `new'
	from main.rb:181:in `run_demo_from_file'
	from main.rb:367:in `block in run_application'
	from .rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/gobject-introspection-3.2.5/lib/gobject-introspection/loader.rb:583:in `invoke'
	from .rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/gobject-introspection-3.2.5/lib/gobject-introspection/loader.rb:583:in `block in define_method'
	from main.rb:518:in `<main>'
Trace/breakpoint trap (コアダンプ)

```
"primary" is below.

https://github.com/ruby-gnome2/ruby-gnome2/blob/8c6d92627869d87ecf3110248e022d33023b49ed/gtk3/sample/gtk-demo/demo.ui#L82

Delete this line then, it works fine.

**Comment by cedlemo** [comment]
Just for information, it works on my system:

* ArchLinux
* ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-linux]
* Gtk3 3.22.30
* ruby gtk3 3.2.6

Could you provide more information on your system ?

**Comment by kojix2** [comment]
Thank you for your reply.

- Ubuntu 16.04
- ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux]
- libgtk-3-0 3.18.9-1ubuntu3.3
- gtk3 (3.2.5)

**Comment by kou** [comment]
I've fixed it.

It's available since GTK+ 3.20.
https://gitlab.gnome.org/GNOME/gtk/commit/9c0e4c174aaae4968bc1970082fdef3183136729

**Comment by kojix2** [comment]
Thank you very much. I confirmed that it works without error.
